web/src/features/weather-layers/solar/stackBlurImageOpacity.ts

```diff
@@ -42,34 +40,39 @@ function BlurStack() {
 
 // Derived from http://www.quasimondo.com/StackBlurForCanvas/StackBlur.js
 export function stackBlurImageOpacity(imageData, top_x, top_y, width, height, radius) {
-  if (isNaN(radius) || radius < 1) return;
-  radius |= 0;
+  if (Number.isNaN(radius) || radius < 1) {
+    return;
+  }
+  radius = Math.trunc(radius);
 
-  var pixels = imageData.data;
+  const pixels = imageData.data;
 
-  var x, y, i, p, yp, yi, yw, a_sum, a_out_sum, a_in_sum, pa, rbs;
+  let x, y, index, p, yp, yi, yw, a_sum, a_out_sum, a_in_sum, pa, rbs;
 
-  var div = radius + radius + 1;
-  var w4 = width << 2;
-  var widthMinus1 = width - 1;
-  var heightMinus1 = height - 1;
-  var radiusPlus1 = radius + 1;
-  var sumFactor = (radiusPlus1 * (radiusPlus1 + 1)) / 2;
+  const div = radius + radius + 1;
+  const w4 = width << 2;
+  const widthMinus1 = width - 1;
+  const heightMinus1 = height - 1;
+  const radiusPlus1 = radius + 1;
+  const sumFactor = (radiusPlus1 * (radiusPlus1 + 1)) / 2;
 
-  var stackStart = new BlurStack();
-  var stack = stackStart;
-  for (i = 1; i < div; i++) {
+  const stackStart = new BlurStack();
+  let stack = stackStart;
+  for (index = 1; index < div; index++) {
     stack = stack.next = new BlurStack();
-    if (i == radiusPlus1) var stackEnd = stack;
+    // eslint-disable-next-line no-var
```

Comment:
Could possibly be removed but I didn't want to mess with the layers too much, they break way too easy and figured it would be best to get the Linting CI up and running before making more changes.